Jenkinsfile

```diff
@@ -8,6 +8,7 @@ node(label: "jenkins-slave") {
     }
     stage("Build") {
       sh 'source rc_ci && make cleanall all'
```

Comment:
I prefer this instead of the `eval ` in the precedent version 👍

Reply:
I don't know why Jenkins did switch to `/bin/sh`